packages/frontend/src/components/badge/Badge.tsx

```diff
@@ -0,0 +1,57 @@
+import cx from 'classnames'
+import React, { ReactNode } from 'react'
+
+export enum BadgeType {
+  WARNING = 'WARNING',
+  ERROR = 'ERROR',
+  GRAY = 'GRAY',
+  BRIGHT_YELLOW = 'BRIGHT_YELLOW',
+  PURPLE = 'PURPLE',
+}
+
+export enum BadgeSize {
+  MD = 'MD',
+  SM = 'SM',
+}
```

Comment:
Maybe we can use Union Types for those? It will result in slightly shorter code, and we'll get type safety anyway.

Reply:
I was also unsure about this one, I think we can do it the way you are talking about.